django-app/donations/payment_gateways/stripe/views.py

```diff
@@ -263,6 +265,43 @@ def cancel_from_stripe(request):
             "Results returned from gateway is invalid."))
     return redirect('donations:cancelled')
 
+@login_required
+def cancel_from_stripe_from_card_update(request):
```

Comment:
probably just `cancel_from_stripe_card_update` is enough? (remove the second "from")
Also remove the second "from" for `return_from_stripe_from_card_update`?